content/use-cases/tokenization/nft-mkt-overview.md

```diff
@@ -76,12 +78,18 @@ Each `NFTokenPage` holds 16-32 NFTs. Minting a large number of NFTs can tie up a
 
 Set up a new wallet. See [Xumm](https://xumm.app/).
 
+<div align="center">
```

Comment:
It doesn't seem like we need an image here as it's a pretty basic concept. Suggest deleting this image.